keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -319,321 +295,20 @@ kind: BucketClass
 metadata:
   name: 
 provisioner: [1]
-supportedProtocols: {"azureblob", "gcs", "s3", ... } [2]
-accessMode: {"ro", "wo", "rw"} [3]
-releasePolicy: {"Delete", "Retain"} [4]
-bucketContentRef: [5]
-  name:
-  uuid:
-secretRef: [6]
-  name:
-  namespace:
+isDefaultBucketClass: [2]
+supportedProtocols: {"azureblob", "gcs", "s3", ... } [3]
+accessMode: {"ro", "wo", "rw"} [4]
+private: boolean [5]
+releasePolicy: {"Delete", "Retain"} [6]
 parameters: [7]
-isDefaultBucketClass: [8]
 ```
 
 1. `provisioner`: The name of the driver. If supplied the driver container and sidecar container are expected to be deployed. If omitted the `secretRef` is required for static provisioning.
+1. `isDefaultBucketClass`: boolean. When true, signals that the COSI controller should attempt to match `Bucket`’s without a defined `BucketClass` to this class, accounting for the `Bucket`’s requested protocol.  Multiple default classes for the same protocol will produce undefined behaviour.
```

Comment:
If `isDefaultBucketClass` is true, I assume this BucketClass is the default for _all_ of the protocols listed.